base/tree.py

```diff
@@ -182,14 +184,16 @@ def timetree(self, Tc=0.01, infer_gtr=True, reroot='best', resolve_polytomies=Tr
             else:
                 node.attr = {'num_date':node.numdate}
             if confidence:
-                node.attr["num_date_confidence"] = sorted(to_numdate(node.marginal_inverse_cdf([0.05,0.95])))
+                node.attr["num_date_confidence"] = sorted(self.tt.get_max_posterior_region(node, fraction=0.9))
```

Comment:
Just by naming I think this is returning the exact same thing. The 90% max posterior will be equivalent to the 5% to 95% bounds on the inverse CDF.